docs/light-client/CHANGELOG.md

```diff
@@ -12,17 +12,26 @@ In this version, all POST methods (except '/tx/broadcast') just generate unsigne
 - Remove POST /tx/sign
 - Remove GET /distribution/community-tax
 - Remove GET /gov/params/{module}
+
+### NON-BREAKING CHANGES:
 - Add Asset module APIs
 - Add Rand module APIs
 - Add Params module APIs
 - Add GET /bank/token-stats/{id}
+- Add POST /bank/accounts/{address}/set-memo-regexp
+
+#### Bank module APIs
+
+| [v0.14.1]      | [v0.15.0]        | input changed | output changed | details | 
+| --------------- | --------------- | --------------- | --------------- | ----- |
+| GET /bank/accounts/{address} | GET /bank/accounts/{address} | No | Yes | Add `memo_regexp` in output, support multiple tokens |
```

Comment:
details --> notes

- Add `memo_regexp` in output;
- Tokens other than `iris-atto` could show up in output when people start using the newly introduced `asset` functionality.